src/sema/contracts.rs

```diff
@@ -733,24 +736,34 @@ fn check_inheritance(contract_no: usize, ns: &mut ast::Namespace) {
     }
 }
 
+#[derive(Default)]
+
+struct ResolveLater<'a> {
+    functions: Vec<(usize, usize, &'a pt::FunctionDefinition)>,
```

Comment:
Instead of this tuple, you could use a struct. This way we know what the first and the second `usize` mean.

Reply:
Yes, you are right. I've added comments to this structure

Reply:
Sorry, I'm introducing structs now